### PR TITLE
Create Rule “update-a-github-issue/rule”

### PR DESCRIPTION
**Pull request by BrookJeynes**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/update-a-github-issue/rule.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+type: rule
+title: Do you to keep a PBI's current status visible?
+uri: keeping-pbis-status-visible
+authors:
+  - title: Brook Jeynes
+    url: https://www.ssw.com.au/people/brook-jeynes/
```
Comment (wicksipedia) [inline rules/update-a-github-issue/rule.md:7]:
```suggestion
  - title: Matt Wicks
    url: https://www.ssw.com.au/people/matt-wicks/
  - title: Brook Jeynes
    url: https://www.ssw.com.au/people/brook-jeynes/
```

File: rules/update-a-github-issue/rule.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+---
+As issues evolve, it's common for their initial descriptions to become outdated or for significant developments to occur that must be recorded for the benefit of the entire team.
+
+<!--endintro-->
+
+Whenever an issue necessitates an update, the team should add a comment to the issue, detailing the change or event. 
+
+::: bad
+
+![Figure: Lots of non-descriptive comments cluttering the issue.](bad-example-lots-of-non-descriptive-comments.png)
```
Comment (wicksipedia) [inline rules/update-a-github-issue/rule.md:22]:
```suggestion
![Figure: Bad example - Lots of non-descriptive comments cluttering the issue.](bad-example-lots-of-non-descriptive-comments.png)
```

File: rules/update-a-github-issue/rule.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+:::
+
+::: good
+
+![Figure: Extra context added via comment.](good-example-adding-context-via-comment.png)
```
Comment (wicksipedia) [inline rules/update-a-github-issue/rule.md:28]:
```suggestion
![Figure: Good example - Extra context added via comment.](good-example-adding-context-via-comment.png)
```

File: rules/update-a-github-issue/rule.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+:::
+
+In cases where an update is long standing or important, it should be appended to the bottom of the issue description. This update must include the date it was made, serving as a chronological record of changes. In addition, a comment should be left on the issue thread to inform team members that significant information has been added to the issue.
+
+This does not include changes regarding to who is currently working on the issue. GitHub and Azure DevOps both track via some version of an “Assignees” tag.
+
+::: good
+
+![Figure: Important information added to the issue itself with a comment.](good-example-update.png)
```
Comment (wicksipedia) [inline rules/update-a-github-issue/rule.md:38]:
```suggestion
![Figure: Good example - Important information added to the issue itself with a comment.](good-example-update.png)
```

File: rules/update-a-github-issue/rule.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+:::
+
+It's also important to CC anyone who may need to see this additional information.
+
+::: good
+
+![Figure: Important users CC'd within the issue but addressed to the team.](good-example-cc-in-issue.png)
```
Comment (wicksipedia) [inline rules/update-a-github-issue/rule.md:46]:
```suggestion
![Figure: Good example - Important users CC'd within the issue but addressed to the team.](good-example-cc-in-issue.png)
```

**Review by JackDevAU — APPROVED** [review]
lgtm! 
Just need to add it to a category, I suggest 1 or both of these,
- https://www.ssw.com.au/rules/rules-to-better-github/
- https://www.ssw.com.au/rules/rules-to-better-scrum-using-github/

File: rules/update-a-github-issue/rule.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+:::
+
+::: good
+
+![Figure: Important users CC'd within comments added extra context.](good-example-cc-in-comment.png)
```
Comment (wicksipedia) [inline rules/update-a-github-issue/rule.md:52]:
```suggestion
![Figure: Good example - Important users CC'd within comments added extra context.](good-example-cc-in-comment.png)
```

File: rules/update-a-github-issue/rule.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+authors:
+  - title: Brook Jeynes
+    url: https://www.ssw.com.au/people/brook-jeynes/
+related:
+  - when-you-use-mentions-in-a-pbi
+  - close-pbis-with-context
+created: 2023-10-03T00:48:22.310Z
+guid: cec9aa77-4234-470b-bcab-557b39de6789
+---
+As issues evolve, it's common for their initial descriptions to become outdated or for significant developments to occur that must be recorded for the benefit of the entire team.
```
Comment (wicksipedia) [inline rules/update-a-github-issue/rule.md:14]:
```suggestion
As PBIs evolve, it's common for their initial descriptions to become outdated or for significant developments to occur that must be recorded for the benefit of the entire team.
```

File: rules/update-a-github-issue/rule.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+<!--endintro-->
+
+Whenever an issue necessitates an update, the team should add a comment to the issue, detailing the change or event.
```
Comment (wicksipedia) [inline rules/update-a-github-issue/rule.md:18]:
```suggestion
Whenever a PBI necessitates an update, the team should add a comment to the issue, detailing the change or event. 
```

**Review by wicksipedia — APPROVED** [review]

**Comment by wicksipedia** [comment]
Hey @JackDevAU 

Noticed the good/bad examples weren't prefixed when you revised the rule.
FYI - For next time, here's the guideline: https://ssw.com.au/rules/use-bad-and-good-examples/

**Pull request merged**